### PR TITLE
JOSS paper update

### PR DESCRIPTION
**Pull request by Ayushk4**
I am opening this PR to JuliaText:josspaper.

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -34,14 +49,23 @@ At present, it contained rules-based methods primarily designed for English.
 Several of the implementations are sourced from the Python NLTK project ([@NLTK1,@NLTK2]);
 although these were in turn source from older pre-existing methods.
 These include the very commonly used Penn Treebank Tokenizer ([@penntok]),
-and the multilingual Tok-tok ([@toktok], [@toktokpub]).
+the multilingual Tok-tok ([@toktok], [@toktokpub]) and tweet tokenizer([@tweettok]).
 Interesting, as an implementation detail, to implement many tokenizers,
 it was found most convenient to first port them to sed,
 and then use Julia's meta-programming capacity to generate Julia expressions from the sed code at compile time.
```
Comment (Ayushk4) [inline paper/paper.md]: We only have `penn_tokenizer` and `improved_penn_tokenizer`, using sedbased approach. Should I change this? What should I change to?

**Comment by Ayushk4** [comment]
@oxinabox Should I include a section on `other software` for natural language tokenization available to Julia users?

File: paper/paper.md [hunk 2]
```diff
@@ -1,47 +1,70 @@
...
 In natural language processing (NLP) tokenization refers to breaking a text up into parts -- the tokens.
 Generally, tokenization refers to breaking a sentence up into words and other tokens such as punctuation.
 Such _word tokenization_ also often includes some normalizing, such as correcting unusual spellings or removing all punctuations.
 Complementary to word tokenization is _sentence segmentation_ (sometimes called _sentence tokenization_),
 where a document is broken up into sentences, which can then be tokenized into words.
-Tokenization and sentence segmentation are some of the most fundamental operations to be performed before applying most NLP or information retreval algorithms
-WordTokenizers.jl exposes a number of tokenization and sentence segmentation functions to allow for this.
+Tokenization and sentence segmentation are some of the most fundamental operations to be performed before applying most NLP or information retrieval algorithms
+WordTokenizers.jl provides a number of fast tokenizers and sentence segmentation functions as well as an API for building custom tokenizers.
```
Comment (oxinabox) [inline paper/paper.md]:
```suggestion
WordTokenizers.jl provides a flexible API for defining fast tokenizers and sentence segementors.
Using this API several standard tokenizers and sentence segmenters have been implemented, allowing researchers and practitioners to focus on the higher details of their NLP tasks.
```

**Comment by oxinabox** [comment]
yes.
It should also mention TextAnalysis and CorpusLoaders as packages that use WordTokenizers.jl

**Comment by Ayushk4** [comment]
It is also being used by https://github.com/chengchingwen/Transformers.jl and BytePairEncoding.jl, should these two also be mentioned?

**Comment by Ayushk4** [comment]
@oxinabox Please review and give suggestions on this.

**Comment by Ayushk4** [comment]
@oxinabox could you review this PR?

**Review by oxinabox — APPROVED** [review]

**Comment by oxinabox** [comment]
Sorry, yes, LGTM

**Pull request merged**